pyproject.toml

```diff
@@ -41,6 +41,9 @@ dependencies    = [
     "ifaddr==0.2.0",
     "Jinja2==3.1.2",
     "lru-dict==1.3.0",
+    "mutagen==1.47.0",
+    # Ensure we run compatible with musllinux build env
+    "numpy==1.26.0",
```

Comment:
numpy is a dependency of multiple integrations, but not directly of core itself?

Where is this used otherwise?

Reply:
`stream` is using numpy. I also experienced this when setting up a new dev env

Reply:
Right, but integration dependencies are set up when those integration set up?

Reply:
See https://github.com/home-assistant/core/pull/81841 and https://github.com/home-assistant/core/pull/81790

Reply:
Yes and no, pre-commit needs them for some reason. I haven't really looked into it as I could fix it, but its an issue reported multiple times

Reply:
Yeah, before the dev environment works completely, you need to start `hass -c config` once and the default_config entry has to be in the configuration.yaml, so that all dependencies get set up.